data/campaigns/tutorial/scenarios/03_Tutorial_part_3.cfg

```diff
@@ -0,0 +1,219 @@
+        no_leader=yes
+        hidden=yes
+
+        [unit]
+            id=Delfador
+            name= _ "Delfador"
+            type=Elder Mage
+            profile=portraits/delfador.png~RIGHT()
+            x,y=13,5
+            random_traits=no
+            facing=nw
+            {IS_HERO}
+            [modifications]
+                {TRAIT_INTELLIGENT}
+                [object]
+                    [effect]
+                        apply_to=new_animation
+                        [extra_anim]
+                            flag=summon_quintain_raise_staff
+                            start_time=0
+                            [frame]
+                                image="units/human-magi/elder-mage-ranged[1~3].png:100"
+                            [/frame]
+                        [/extra_anim]
+                        [extra_anim]
+                            flag=summon_quintain_lower_staff
+                            start_time=0
+                            [frame]
+                                image="units/human-magi/elder-mage-ranged[3~1].png:150"
+                            [/frame]
+                        [/extra_anim]
+                    [/effect]
+                [/object]
+            [/modifications]
+        [/unit]
+    [/side]
+
+    [event]
+        name=prestart
+        [objectives]
+            side=1
+            [objective]
+                description= _ "End your turn (end tutorial)"
+                condition=win
+            [/objective]
+
+            [objective]
+                description= _ "Move Konrad to a signpost"
+                condition=win
+                [show_if]
+                    [have_unit]
+                        id=student
+                        gender=male
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+
+            [objective]
+                description= _ "Move Li’sar to a signpost"
+                condition=win
+                [show_if]
+                    [have_unit]
+                        id=student
+                        gender=female
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+
+            [note]
+                description= _ "Each signpost has a longer description — right-click on the signpost and select “describe signpost” to read it."
+            [/note]
+        [/objectives]
+
+        # In case a [set_menu_item] is added without the corresponding [clear_menu_item],
+        # ensure that these menu items are only displayed in this scenario.
+        [unit]
+            id=the_s3_quintain
+            type=Quintain
+            ai_special=guardian
+            x,y=18,5
+            side=2
+        [/unit]
+#define SHOW_MENU_ITEM_ON_S3_ONLY
+        [show_if]
+            [have_unit]
+                id=the_s3_quintain
+            [/have_unit]
+        [/show_if]
+#enddef
+
+        {PLACE_IMAGE "scenery/signpost.png" 6 3}
+        {SET_LABEL 6 3 _"Replay Elves vs Orcs"}
```

Comment:
So it's a minor thing, but because it's the tutorial and all new players will go through here, I'll ask it: should we add a signpost labeled "Exit to title screen", so it's clear that's an option too?